### PR TITLE
SDK version bump

### PR DESCRIPTION
**Pull request by Deepak-Kesavan**
## What

SDK version bump

## Why

-

## How

-

## Database Migrations

- 

## Env Config

- 

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: backend/pyproject.toml [hunk 1]
```diff
@@ -30,7 +30,7 @@ dependencies = [
     "python-socketio==5.9.0", # For log_events
     "social-auth-app-django==5.3.0", # For OAuth
     "social-auth-core==4.4.2", # For OAuth
-    "unstract-sdk~=0.20.0",
+    "unstract-sdk~=0.20.1",
```
Comment (gaya3-zipstack) [inline backend/pyproject.toml:33]: Patch version gets automatically picked up as long the dependency is explicitly included

**Review by chandrasekharan-zipstack — APPROVED** [review]
LGTM - make sure the cloud related version updates are performed too

File: backend/pyproject.toml (hunk 1 above)
Comment (Deepak-Kesavan) [reply backend/pyproject.toml:33]: Yes @gaya3-zipstack . Just added it to track SDK version in tool.

File: tools/structure/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # Add your dependencies here
 
 # Required for all unstract tools
-unstract-sdk~=0.20.0
+unstract-sdk~=0.20.1
```
Comment (gaya3-zipstack) [inline tools/structure/requirements.txt:4]: Tools need to be built. FYI....

Comment (Deepak-Kesavan) [reply tools/structure/requirements.txt:4]: Yes

**Review by gaya3-zipstack — APPROVED** [review]
Looks good. Minor info added for next time.

**Pull request merged**